native/NativeSampleApps/RestExplorer/res/values/bootconfig.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <resources>
-    <string name="remoteAccessConsumerKey">3MVG9Iu66FKeHhINkB1l7xt7kR8czFcCTUhgoA8Ol2Ltf1eYHOU4SqQRSEitYFDUpqRWcoQ2.dBv_a1Dyu5xa</string>
-    <string name="oauthRedirectURI">testsfdc:///mobilesdk/detect/oauth/done</string>
+    <string name="remoteAccessConsumerKey">3MVG94DzwlYDSHS5d9A2_Ra7sqI.2ttBQuZPF1cS65n9QOZFzBL0F36mCZe33SMzKDKXFthjUNvbJMo.faevi</string>
+    <string name="oauthRedirectURI">spsfdc:///mobilesdk/detect/oauth/done</string>
```

Comment:
What connected app is this?

Reply:
A new one. IDP/SP flow only works if the two apps use different consumer keys. I think I will revert that change here and have the IDP template use a new connected app.